stubs/pyflakes/pyflakes/reporter.pyi

```diff
@@ -1,5 +1,9 @@
+from typing import TextIO
+
+from .messages import Message
+
 class Reporter:
-    def __init__(self, warningStream, errorStream) -> None: ...
-    def unexpectedError(self, filename, msg) -> None: ...
-    def syntaxError(self, filename, msg, lineno, offset, text) -> None: ...
-    def flake(self, message) -> None: ...
+    def __init__(self, warningStream: TextIO, errorStream: TextIO) -> None: ...
```

Comment:
`TextIO` is not a great type for stubs. I have a personal vendetta against it and its related types `BinaryIO` and `IO`. Here, `_typeshed.SupportsWrite` (or in the future `typing_extensions.Writer`) is a better option:

```suggestion
    def __init__(self, warningStream: SupportsWrite[str], errorStream: SupportsWrite[str]) -> None: ...
```